dumper/README.md

```diff
@@ -0,0 +1,16 @@
+# BigQuery Migration Service Metadata and Log Dumper
+
+This directory contains the Metadata and Log Dumper, a command line tool for
+connecting to an existing database and generating an archive of DDL metadata or
+logs. This tool generates archives in a format suitable for consumption by the
+[BigQuery Migration Service's][BQMS] Assessment or Translation Service.
+
+The Dumper is a Java tool. **Download the latest cross-platform release zip
+`dwh-migration-tools-vX.X.X.zip` from
+[the Releases page](https://github.com/google/dwh-migration-tools/releases).**
```

Comment:
We can point to latest release: https://github.com/google/dwh-migration-tools/releases/latest

Reply:
I'm ambivalent pointing users at a page that only shows the latest release, but I went ahead and changed all the links to do this since you can get the full list of releases from the breadcrumbs bar on that page anyway if you need it.